tools/myst/myst.edl

```diff
@@ -513,5 +513,13 @@ enclave
             [out] uint32_t* rbx,
             [out] uint32_t* rcx,
             [out] uint32_t* rdx);
+        
+        long myst_get_mempolicy_ocall(
+            [out] int* mode,
+            [out, size=maxnode] unsigned long* nodemask,
+            unsigned long maxnode,
+            [in] void* addr,
```

Comment:
EDL `in` syntax instruct the OCALL framework to copy a buffer referenced by a pointer from inside the enclave to outside of the enclave, so the outside enclave code can deference the pointer to the buffer (outside of the enlcave) and access the buffer. In get_mempolicy syscall, `addr` pointer will not be dereferenced. The eDL should not have `[in]`

Reply:
Got it, updated this. Will out be accurate then in this case? Earlier comment says `should not have [out]`